node/node_explorer.go

```diff
@@ -177,3 +177,34 @@ func (node *Node) GetTransactionsHistory(address, txType, order string) ([]commo
 	}
 	return hashes, nil
 }
+
+// GetStakingTransactionsHistory returns list of staking transactions hashes of address.
+func (node *Node) GetStakingTransactionsHistory(address, txType, order string) ([]common.Hash, error) {
+	addressData := &explorer.Address{}
+	key := explorer.GetAddressKey(address)
+	bytes, err := explorer.GetStorageInstance(node.SelfPeer.IP, node.SelfPeer.Port, false).GetDB().Get([]byte(key), nil)
+	if err != nil {
+		return make([]common.Hash, 0), nil
```

Comment:
any specific reason for ignoring the error and returning empty array of hashes?

Reply:
hmmm this code is same as GetTransactionsHistory() above. I can add logger error here.